portfolio/src/main/webapp/events.html

```diff
@@ -6,6 +6,7 @@
         <link href="https://fonts.googleapis.com/css2?family=Titillium+Web&display=swap" rel="stylesheet"> 
         <link rel="stylesheet" href="events.css">
         <script src="events.js"></script>
+        <script src="script.js"></script>
```

Comment:
is this declaration of script.js still necessary? looks like the file was removed